pkg/minikube/tunnel/registry.go

```diff
@@ -61,7 +61,10 @@ func (r *persistentRegistry) IsAlreadyDefinedAndRunning(tunnel *ID) (*ID, error)
 	}
 
 	for _, t := range tunnels {
-		if t.Route.Equal(tunnel.Route) {
+		// it is possible that different tunnel can be started for different tunnel
```

Comment:
comment on the t.route needs more clear example why we need that

Reply:
If you had a list of steps to explain how this is reproduced that would be great

Reply:
@ComradeProgrammer please consider making this t.Route Change in a separate PR so we make this PR only related to the Newest Version Test